src/Uno.Extensions.Validation.Fluent/Uno.Extensions.Validation.Fluent.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+	<Import Project="..\tfms-non-ui.props" />
+
+	<PropertyGroup>
+		<Description>FluentValidation extensions for Uno Platform, UWP and WinUI</Description>
+
+		<!--Temporary disable missing XML doc until fixed in the whole package-->
+		<WarningsNotAsErrors>$(WarningsNotAsErrors);CS1591</WarningsNotAsErrors>
+
+		<!-- As we are InternalsVisibleTo, we disable some compatibility types that are not used by package itself. -->
+		<UnoExtensionsGeneration_DisableModuleInitializerAttribute>True</UnoExtensionsGeneration_DisableModuleInitializerAttribute>
```

Comment:
```suggestion
		<!-- As we are InternalsVisibleTo, we disable some compatibility types that are not used by package itself. -->
		<UnoExtensionsGeneration_DisableModuleInitializerAttribute>True</UnoExtensionsGeneration_DisableModuleInitializerAttribute>
```

You should be able to remove this also. Insteed we should remove the `InternalsVisibleTo` if needed.